### PR TITLE
BW-1374: CBAS uses helm postgres

### PR DESCRIPTION
**Pull request by cjllanwarne**
Coordinate merging with https://github.com/DataBiosphere/cbas/pull/36

### CODE REVIEW

File: cromwell-helm/values.yaml [hunk 1]
```diff
@@ -148,7 +150,8 @@ cbasUI:
 
 cbas:
   name: cbas
-  image: us.gcr.io/broad-dsp-gcr-public/cbas:0.0.19
+  # TODO: REVERT!
+  image: broadinstitute/cbas:0.0.20-cjl-test
```
Comment (cjllanwarne) [inline cromwell-helm/values.yaml:154]: Tsk...

**Review by kpierre13 — APPROVED** [review]

**Review by mspector — APPROVED** [review]

File: terra-batch-libchart/templates/_cbas-api.tpl [hunk 2]
```diff
@@ -77,6 +77,19 @@ data:
       healthcheckEndpoint: "/swagger/swagger-ui.html"
       instanceId: "15f36863-30a5-4cab-91f7-52be439f1175"
       apiV: "v0.2"
+    spring:
```
[the remark this answers is not in the document]
Comment (jgainerdewar) [reply terra-batch-libchart/templates/_cbas-api.tpl:80]: Can these strings be templated at all? Can we define chunks that will vary across the two charts and define them in separate variables, then insert them in the appropriate places in this string in the library chart?

[the remark this answers is not in the document]
Comment (mspector) [reply terra-batch-libchart/templates/_cbas-api.tpl:80]: ah, yes, that is a good idea.

(in fact, I used it in my recent refactor in the cromwell config - I don't know why I didn't suggest that myself)

[the remark this answers is not in the document]
Comment (mspector) [reply terra-batch-libchart/templates/_cbas-api.tpl:80]: see: https://github.com/broadinstitute/cromwhelm/pull/68/files#diff-11c714efcd3e99fa0bac91d4ff1f86d687ab4e48e387df0c63f5eec218ffe022R135

**Pull request merged**